odoo-openmrs/src/main/java/com/ozonehis/eip/odoo/openmrs/client/OdooUtils.java

```diff
@@ -84,4 +87,17 @@ private Object convertValueToType(Object value, Class<?> targetType) {
         }
         throw new IllegalArgumentException("Unsupported type: " + targetType.getName());
     }
+
+    public static String convertEEE_MMM_ddDateToOdooFormat(String date) {
```

Comment:
It's anti-pattern to use underscores in the Java method names
```suggestion
    public static String convertToOdooDateFormat(String date) {
```

Reply:
Wanted to keep this method name generic by specifying the date format and not saying `OdooDateFormat`
Will rename it to `convertEEEMMMddDateToOdooFormat`